### DISCUSSION TITLE
Typo in TestScheduler.ts

### DISCUSSION THREAD

**Discussion by wildcardalice** [ReactiveX/rxjs#6409] [Report issues other than bug]
Just a heads up, there's a typo here (twice on the same file): https://github.com/ReactiveX/rxjs/blob/master/src/internal/testing/TestScheduler.ts#L86

"Ommitted" -> "Omitted"

**Comment by cartant** [comment]
PRs that fix typos - however small - are always welcome and will be merged with a "thank you." 🙂
